ovve_ui.py

```diff
@@ -64,19 +67,20 @@ def updateValue(self, value):
 
 
 class SimpleDisplayButton(QAbstractButton):
-    def __init__(self, value, parent=None, size=(200, 50)):
+    def __init__(self, value, button_settings, parent=None, size=(200, 50)):
```

Comment:
Can button_settings also contain the size?

Reply:
My intention was that the button_settings were unified across the entire UI to allow for styling all boxes at once. That was why I omitted size.